### PR TITLE
RHTAPINST-68: Red Hat Quay

### PR DESCRIPTION
**Pull request by otaviof**
Quay operator support plus a few other fixes.

### CODE REVIEW

File: charts/rhtap-backing-services/templates/openshift-quay/_quay.tpl [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{{- define "backingServices.quay.configSecretName" -}}
+  {{- printf "%s-config" .Values.backingServices.quay.name }}
+{{- end -}}
+
+{{/*
+
+  Quay "quay_config.yaml" file template. Uses the cofigured S3 storage credentials
```
Comment (Roming22) [inline charts/rhtap-backing-services/templates/openshift-quay/_quay.tpl]: cofigured -> configured

File: charts/rhtap-infrastructure/values.yaml [hunk 2]
```diff
@@ -113,6 +111,40 @@ infrastructure:
               resources:
                 requests:
                   storage: 500Mi
+    #
+    # Quay.io Tentant, stores the container image artifacts.
```
Comment (Roming22) [inline charts/rhtap-infrastructure/values.yaml]: Tentant -> Tenant

File: charts/rhtap-infrastructure/values.yaml [hunk 3]
```diff
@@ -113,6 +111,40 @@ infrastructure:
...
+    #
+    quay:
+      enabled: false
+      # Target namespace for the MinIO Tenant and related resources.
+      namespace: __OVERWRITE_ME__
+      # Secret name to store MinIO's root passord, the password is genearted when
```
Comment (Roming22) [inline charts/rhtap-infrastructure/values.yaml]: passord -> password
genearted -> generated

Comment (otaviof) [reply charts/rhtap-infrastructure/values.yaml]: Thanks.

File: charts/rhtap-infrastructure/values.yaml (hunk 2 above)
Comment (otaviof) [reply charts/rhtap-infrastructure/values.yaml]: Thanks.

**Review by Roming22 — APPROVED** [review]
Can be merged as soon as the CI is green.

**Comment by otaviof** [comment]
> Can be merged as soon as the CI is green.

Last issue we've found should be fixed.

**Pull request merged**